modules/administration/pages/security-hardening.adoc

```diff
@@ -0,0 +1,54 @@
+[[security-hardening]]
+= Security Hardening
+
+While no implementation is ever completely secure from bad actors, there are some things you can do within {productname} to improve your security.
+
+This section contains a number of suggestions for hardening your security posture.
+The primary focus is to reduce the likelihood that a bad actor could access Salt clients in the case of a breach of the {productname} Server.
+In this case, we assume that the Salt master is running on the {productname} Server.
+
+To begin, there are some built-in mechanisms that make the {productname} Server less vulnerable to bad actors.
+The Salt master runs as a non-privileged user, and the Salt API only listens on localhost, not an external address.
+
+Additionally, Salt clients cannot send jobs to the Salt master in the default configuration.
+This can be manually adjusted, but it is not recommended.
+
+There are a limited number of ports required for clients to communicate with the server, which simplifies network management.
+The only required ports are 4505 and 4506.
+Ports 80 and 443 are only used for package repositories, and they secured with SSL trusted certificates and GPG keys.
+
+The server never initiates a TCP handshake with a client.
+Clients initiate every connection, and the communication is bi-drectional after initiation.
```

Comment:
```suggestion
Clients initiate every connection, and the communication is bi-directional after initiation.
```

Reply:
Exception: salt-ssh or ssh-push . Here the server initiate a ssh connection to the client.